lib/changelog/schema/episode/episode.ex

```diff
@@ -385,6 +385,8 @@ defmodule Changelog.Episode do
         flatten_episode_for_filtering(episode)
       end)
 
+    # TODO: This is causing some very slow start up times
+    # Probably needs some investigation and optimization
```

Comment:
This is likely the same thing I'm experiencing where loading up the EpisodeTracker on boot is slow. Yeah?

Reply:
Yes I think so. I actually had to turn up the Ecto timeout on my MacBook air as the app wouldn't start otherwise due to how long it took to get through that transaction.